Fix exponentiated gradient citation

audreyfeldroy opened the pull request:
## Description
During the PyLadiesCon fastlearn sprint, @TamaraAtanasoska and I paired on fixing the broken citation on the [fairlearn.reductions.ExponentiatedGradient](https://fairlearn.org/v0.11/api_reference/generated/fairlearn.reductions.ExponentiatedGradient.html#fairlearn.reductions.ExponentiatedGradient) page.

## Tests
<!--- Select all that apply by putting an x between the brackets: [x] -->
- [x] no new tests required
- [ ] new tests added
- [ ] existing tests adjusted

## Documentation
<!--- Select all that apply. -->
- [ ] no documentation changes needed
- [ ] user guide added or updated
- [X] API docs added or updated
- [ ] example notebook added or updated

TamaraAtanasoska commented:
@audreyfeldroy congrats on the first contribution 🎉 ! we forgot the linting and the `pre-commit hooks` (my bad!). They are in the "development process" [page](https://fairlearn.org/main/contributor_guide/development_process.html), but not in the "contributing docs" guide that we used. You can install the `pre-commit hooks` for future contributions, but right now you can just run the `black` command on the file you changed and push the reformatted file to make the linting test that is currently failing pass.

audreyfeldroy commented:
Thanks @TamaraAtanasoska! Fortunately my daughter is sleeping in more than usual for a Sunday morning, so I have a few minutes 😁 I've now:
1. Formatted the changed file with black
2. Rebased the PR, dropping the accidental unrelated commit
3. Installed pre-commit locally
4. Updated this branch

hildeweerts commented:
Awesome, thank you @audreyfeldroy (and your daughter 😄!)
